scripts/reana

```diff
@@ -104,6 +106,91 @@ down () {
+    flag_verbose=$(echo "$*" | grep -e "--verbose" -o)
+    flag_devel=$(echo "$*" | grep -e "--devel" -o)
+
+    if [ "$demonstration" != "" ]; then
+        #Check that there exists such a demo as inputted by user.
+        if echo "$reana_demos" | grep -qE "^$demonstration$"
+            then
+            # FIXME: Check REANA cluster is accessible. Simple ping/pong, or interaction with kubectl?
+
+            # Get URL of reana-workflow-controller
+            controller_url=$(reana get reana-workflow-monitor)
+
+            # Submit a helloworld-workflow job to workflow-controller
+            workflow_id=$(curl --fail --silent \
+                            -H "Content-Type: application/json" \
+                            -X POST -d \
+                            "{\"experiment\":\"atlas\",
+                              \"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\",
+                              \"workflow\": \"hello.yaml\",
+                              \"nparallel\": \"10\",
+                              \"preset_pars\": {\"name\": \"J.Doe\", \"delay\": \"2\"}
+                            }" \
+                            http://"$(reana get reana-workflow-controller)"/yadage \
+                            | grep -oP '(?<="workflow_id": ")[^"]*')
+
+            # Check for successful execution of curl
+            # https://unix.stackexchange.com/a/204790
+            # Note that checking with '$?' might fail on some corner cases
+            # AND when authentication is involved (response codes 401 and 407)
+            submit_status=$?
+
+            if [ "$submit_status" = "0" ]; then
+
+                if [ "$flag_verbose" != "" ]; then
+                    echo "[INFO] Open http://$controller_url/$workflow_id to see visual representation of your workflow and it's progress."
+                    echo "[INFO] You can 'cat' or 'tail' /reana/atlas/$workflow_id/yadage/helloworld/greetings.txt -file"
+                    echo "       from shared storage of REANA cluster to see how helloworld workflow progresses."
+                    echo "[INFO] (e.g. minikube ssh \"tail -c+1 -F /reana/atlas/$workflow_id/yadage/helloworld/greetings.txt\")"
+                fi
+
+                echo ""
+                echo "Workflow_id: $workflow_id"
+                echo "Monitor: http://$controller_url/$workflow_id"
+                echo "Output:  /reana/atlas/$workflow_id/yadage/helloworld/greetings.txt"
+
+                if [ "$flag_devel" != "" ]; then
+                    # Start tailing of job-controller logs
+                    #kubectl logs -f $(kubectl get pods | grep -Po 'job-controller[^ ]+')
+
+                    # Start tailing of yadage-atlas-worker logs
+                    #kubectl logs -f $(kubectl get pods | grep -Po 'yadage-atlas-worker[^ ]+')
+
+                    # Open workflow-monitor web-interface
+                    firefox "http://$controller_url/$workflow_id"
```

Comment:
It would be good to take commands like `firefox` out of the script (and use them only in docs).